### PR TITLE
Add leaflet.controlgeocoder

### PR DESCRIPTION
**Pull request by Kcchouette**
From https://github.com/perliedman/leaflet-control-geocoder

Thanks

### CODE REVIEW

File: files/leaflet.controlgeocoder/update.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "packageManager": "npm",
+  "name": "platform",
```
Comment (jimaek) [inline files/leaflet.controlgeocoder/update.json]: You are pointing here https://www.npmjs.com/package/platform
This is incorrect

Comment (Kcchouette) [inline files/leaflet.controlgeocoder/update.json]: @jimaek corrected; thanks :)

File: files/leaflet.controlgeocoder/update.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "packageManager": "npm",
+  "name": "leaflet-control-geocoder",
+  "files": {
+    "include": ["Control.Geocoder.css", "Control.Geocoder.js"]
```
Comment (jimaek) [inline files/leaflet.controlgeocoder/update.json]: There is no js file there:
```
[root@ProspectOne ~]# ll node_modules/leaflet-control-geocoder/
total 76
-rw-r--r-- 1 root root   804 Nov  6 09:02 bower.json
-rw-r--r-- 1 root root  2292 Aug 21 08:42 Control.Geocoder.css
drwxr-xr-x 2 root root  4096 Oct 16 09:14 demo
drwxr-xr-x 3 root root  4096 Jan 13 12:30 dist
-rw-r--r-- 1 root root 10683 Aug 21 10:42 docs.html
drwxr-xr-x 2 root root  4096 Jan 13 12:30 images
-rw-r--r-- 1 root root  1362 Aug 21 09:12 LICENSE
drwxr-xr-x 3 root root  4096 Jan 13 12:30 node_modules
-rw-r--r-- 1 root root 10032 Jan 13 12:30 package.json
-rw-r--r-- 1 root root  8503 Oct 16 09:14 README.md
drwxr-xr-x 2 root root  4096 Jan 13 12:30 scripts
drwxr-xr-x 3 root root  4096 Jan 13 12:30 src
drwxr-xr-x 2 root root  4096 Jan 13 12:30 test

```

**Comment by Kcchouette** [comment]
Thanks for your help.
Hope this fix all :)

**Comment by Kcchouette** [comment]
Can you add it, please?

**Pull request merged**